docs/source/livemode.md

```diff
@@ -0,0 +1,87 @@
+# Livemode - having a more familiar, possibly interactive, upgrade environment
+**Note that the livemode feature is experimental.**
+
+Leapp features an experimental feature named _livemode_ that partially replaces
```

Comment:
The sentense is kind of hard to get, let's simplify it to something like

```suggestion
_livemode_ is an experimental feature that partially replaces
```

Reply:
Suggestion applied :+1: